File: Quiz1/PowerPangers/Series.cs
using System.Drawing;

namespace Quiz1.PowerPangers;

public class Series
{
    public void RunSeries()
    {
        var monster = new Monster();

        var heroes = new[]
        {
            new Hero(Color.Blue),
            new Hero(Color.Red),
            new Hero(Color.Yellow),
            new Hero(Color.Black),
            new Hero(Color.Pink)
        };
        var heroCommand = new HeroCommand(heroes);

        heroCommand.Kill(monster);

        monster.SetBig();

        heroCommand.GetZords();

        heroCommand.Kill(monster);
    }
}

public class Monster
{
    public void SetBig()
    {
    }
}

public class Hero
{
    private readonly Color _color;

    public Hero(Color color)
    {
        _color = color;
    }
}

public class HeroCommand
{
    private readonly Hero[] _command;
    private Hero[] command;

    public HeroCommand(Hero[] command)
    {
        _command = command;
    }

    public void Kill(Monster monster)
    {
    }

    public void GetZords()
    {
    }
}

File: Quiz1/PowerPangers/Series.java
import java.awt.Color;

public class Series {
    public void runSeries() {
        Monster monster = new Monster();

        Hero[] heroes = {
            new Hero(Color.BLUE),
            new Hero(Color.RED),
            new Hero(Color.YELLOW),
            new Hero(Color.BLACK),
            new Hero(Color.PINK)
        };
        HeroCommand heroCommand = new HeroCommand(heroes);

        heroCommand.kill(monster);

        monster.setBig();

        heroCommand.getZords();

        heroCommand.kill(monster);
    }
}

class Monster {
    public void setBig() {
    }
}

class Hero {
    private final Color color;

    public Hero(Color color) {
        this.color = color;
    }
}

class HeroCommand {
    private final Hero[] command;

    public HeroCommand(Hero[] command) {
        this.command = command;
    }

    public void kill(Monster monster) {
    }

    public void getZords() {
    }
}